osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -130,7 +130,6 @@ private void load(AudioManager audio, IdleTracker idleTracker, GameHost host)
 
             buttonsTopLevel.Add(new Button(@"play", @"button-play-select", OsuIcon.Logo, new Color4(102, 68, 204, 255), () => State = ButtonSystemState.Play, WEDGE_WIDTH, Key.P));
             buttonsTopLevel.Add(new Button(@"osu!editor", @"button-generic-select", OsuIcon.EditCircle, new Color4(238, 170, 0, 255), () => OnEdit?.Invoke(), 0, Key.E));
-            buttonsTopLevel.Add(new Button(@"osu!direct", @"button-direct-select", OsuIcon.ChevronDownCircle, new Color4(165, 204, 0, 255), () => OnBeatmapListing?.Invoke(), 0, Key.D));
```

Comment:
Should probably trim the `OnBeatmapListing` action too, unless we're going to be bringing it back at a later date.